packages/server/src/service/static/pug/embed.pug

```diff
@@ -0,0 +1,224 @@
+extends layout.pug
```

Comment:
![image](https://user-images.githubusercontent.com/1906321/91215649-a89e8f00-e6c9-11ea-8db8-0ed730725a4c.png)

Let's simplify this page and improve UX.
1. Clicking the buttons doesn't do anything.  You should just put the HTML code into a disabled text box and let users copy from it.  That's the most clear UX.
2. We only need flexible dimensions -- responsive is basically always good and is a good default.  If they want something fancier, they should hire someone.  We don't need any of the other boxes -- just the bottom right.  Let's put this on the details page of each landing page so that the landing page URL comes for free.